src/modm/driver/can/mcp2515_impl.hpp

```diff
@@ -231,79 +299,127 @@ modm::Mcp2515<SPI, CS, INT>::getMessage(can::Message& message)
 
 	// RX0IF or RX1IF respectivly were already cleared automatically by rising CS.
 	// See section 12.4 in datasheet.
-
 	return true;
 }
 
+template <typename SPI, typename CS, typename INT>
+modm::ResumableResult<bool>
```

Comment:
Shouldn't `update()` return `modm::ResumableResult<void>`?
Because nobody (in the main loop) will handle the return value...

Reply:
it's just the return value of sendMessage indicating if a message was sent this cycle ... not really useful but i thought why not :D ... iwill remove it

Reply:
it is void now